pkg/controllermanager/controller/shootstate/add_test.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package shootstate_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/controllermanager/controller/shootstate"
+)
+
+var _ = Describe("AddToManager", func() {
```

Comment:
For consistency with other similar unit tests.

```suggestion
var _ = Describe("Add", func() {
```

Reply:
Thanks for catching this mismatched naming, fixed it in my latest updates.